README.md

```diff
@@ -217,6 +217,17 @@ sudo /home/pi/fbcp-ili9341/build/fbcp-ili9341 &
 
 If the user name of your Raspberry Pi installation is something else than the default `pi`, change the directory accordingly to point to the user's home directory. (Use `pwd` to find out the current directory in terminal)
 
+##### Launching the display driver with `systemd`
+
+Alternatively, instead of modifying `/etc/rc.local`, use the provided `systemd` unit file as below:
+
+```bash
+sudo install -m 0644 -t /etc fbcp.conf 
+sudo install -m 0644 -t /etc/systemd/system fbcp.service 
+sudo systemctl daemon-reload
+sudo systemctl enable fbcp && sudo systemctl start fbcp

```

Comment:
Thanks for the PR! Could the name be renamed from `fbcp` to `fbcp-ili9341` to keep with the repository name?

Reply:
done